bin/fresh-node10

```diff
@@ -1,96 +1,48 @@
 #!/bin/bash -u
```

Comment:
Just thinking through some probable usage scenarios (user already has local unsafe npm installed, user might not always remember new fresh-node workflow) would it be worthwhile to include a shell alias snippet like this with fresh-node?  Or in the README, perhaps?

```
npm_check() {
    echo "Are you sure you want to run an UNSANDBOXED version of npm on this machine?"
    echo -e "Did you intend to use fresh-node instead?\n"
    read -p "[Y]es or [N]o: " -n 1 -r
    if [[ ! $REPLY =~ ^[Yy]$ ]]
    then
        echo -e "\n\nNow executing potentially insecure npm from local machine...\n"
        $(which npm) $@
    else
        echo
        fresh-node $@
    fi  
}
alias npm=npm_check
```

Reply:
Yep, this makes a lot of sense. I was thinking of going a step further and enforcing it from the side of the npm run scripts. E.g. in `mediawiki-core/package.json` we'd bail out (or ask to confirm) if run in a way that looks unsafe.

But, I like this approach as well.